docs/docs/Configuration.md

```diff
@@ -3,7 +3,7 @@ title: "Configuration Parameters"
 linkTitle: "Configuration"
 weight: 5
 description: >
-    RedisBloom supports multiple module configuration parameters. All of these parameters can only be set at load-time.
```

Comment:
When we talk about the HOW to configure, we still need to use the module name. RedisBloom